content/updates/2020-12/2-Revamped-data-quality-dashboard.md

```diff
@@ -0,0 +1,23 @@
+---
+pim_announcement_alt_img: "The revamped DQI dashboard"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# A revamped data quality dashboard
+::: meta-data type="Improvement" features="Quality" available="early December" link-to-doc="../articles/improve-data-quality.html#know-your-data-quality"
+
+Under the `Activity` menu, discover a totally reworked `Data Quality Insights` dashboard to easily follow the catalog's data quality. Besides the UX improvement, a new chart and key indicators are here to help you get a better overview of where you are and the progress you made so far on this crucial topic: the quality of your product information. 
+
+![The revamped DQI dashboard](../img/new-dqi-dashboard.png) 
+
+A step chart now shows the overall quality score evolution on a monthly basis.  
+New key indicators are also displayed to draw your attention to specific quality criteria, such as the percentage of products without images. Those indicators being taken into account in the quality score, don't hesitate to use them to drive your product enrichment and reach a better quality score.
```

Comment:
Should we specify that "products without images" relate to asset attributes?
```suggestion
New key indicators are also displayed to draw your attention to specific quality criteria, such as the percentage of products without images. Those indicators are taken into account in the quality score, don't hesitate to use them to drive your product enrichment and reach a better quality score.
```

Reply:
Not only asset attributes, but also basic image attribute. ;)